dds_registration/models.py

```diff
@@ -85,6 +85,24 @@ def __init__(self, *args, **kwargs):
         self._original_username = self.username
 
 
+class Membership(models.Model):
+    user = models.ForeignKey(User, related_name='registrations', on_delete=models.CASCADE)
```

Comment:
I was asked to use a different related name. I've used `related_name='memberships'`.

Reply:
👍🏽